app/views/fragments/promotion/subscribeNow.scala.html

```diff
@@ -0,0 +1,10 @@
+@import model.DigitalEdition
+@import com.gu.memsub.promo.PromoCode
+@import views.support.DigitalEdition._
+@(edition: DigitalEdition, promoCode: PromoCode)
+<div class="pricing-cta__action">
```

Comment:
Since `__action` is BEM child syntax, I'd expect this to be within a `.pricing-cta` element